src/api.lisp

```diff
@@ -4,7 +4,9 @@
 
 (defgeneric error-map-type (error-map))
 
-(defmacro define-error-map (name error-type no-error bindings)
+(defgeneric error-map-fatal-code (error-map))
+
+(defmacro define-error-map (name error-type (&key no-error fatal-error) bindings)
```

Comment:
It's a little unfortunate that this breaks extant call sites of the macro.

Define a new macro with a new name and the desired lambda list, perhaps calling this macro in its body, and document this macro as deprecated?

Reply:
I think we're the only users of this. We can just update our usage internally when this lands.